learing_journal/security_lecture.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
This entire file is sample code.  It should not be in your repository.  Please delete it so that there can be no confusion about which files we should be looking at for grading.